db/migrate/20160524231157_add_verified_to_identities.rb

```diff
@@ -0,0 +1,7 @@
+class AddVerifiedToIdentities < ActiveRecord::Migration
+  def change
+    add_column :identities, :verified, :boolean, default: false
+
+    add_index :identities, :verified
+  end
+end
```

Comment:
Let's update this; we need to support multiple Identity Assurance Levels -- change to IAL